ingestion-edge/docker-compose.yml

```diff
@@ -38,15 +38,12 @@ services:
       ]
   test:
     image: *image
-    entrypoint: py.test
-    command:
-    - --flake8
-    - --docstyle
-    - --mypy
-    - --mypy-ignore-missing-imports
+    entrypoint: bin/test
     depends_on:
     - pubsub
     - web
+    env_file:
+    - docker-compose.test.env
```

Comment:
I think we should leave this as part of the circle config, or make it `env: {CIRCLE: "${CIRCLE}"}` so that this file doesn't get applied when locally running `docker-compose` after locally running `circleci build --job ingestion-edge`

Reply:
Refactored to have a docker-compose.circleci.env that contains all the changes specific to running in CI